index.rst

```diff
@@ -326,33 +326,167 @@ Analysis of unbinned data is clearly needed for pipeline data quality analyses,
-Suggestions to create new individualized databases should be resisted unless no existing database can be utilized.
-Whereas much of the data coming off the summit is time based and therefore goes into a time-based database (the EFD), other aspects of the system are image based, such as what will be produced by Rapid Analysis and the parts of the camera system.
-The implementation of various project databases is currently being discussed and documented in a number of tech notes [#]_ however, the capabilities and functionalities required by the commissioning team has not been explicitly described.
+We revisit the need for a Visit database at the Summit.
+This relational database consolidates raw image metadata, Rapid Analysis outputs, telemetry data, and other datasets related to the observations.
 
-.. [#] For further details, consult the following technotes, which are in various states of being written: `Sasquatch`_, the `Butler <https://dmtn-204.lsst.io>`_,  database support for `campaigns <https://dmtn-220.lsst.io/>`_, as well as the `consolidated database <https://dmtn-227.lsst.io/>`_.
+We discuss possible paths for its implementation using existing tools in DM, bearing in mind the near real-time feedback requirements of Rapid Analysis.
 
-FAFF is assembling a series of use-cases, specifically descriptions of database queries, that will identify the commissioning-specific functionalities required by the project databases.
-This content is currently hosted on `a confluence page <https://confluence.lsstcorp.org/display/LSSTCOM/Use+cases+for+commissioning+databases>`_, but the pertinent content will be merged to this report and/or the use-cases described as part of `Deliverable 1`_.
+In addition to the Visit database, we recommend recording scalar metrics into a time series database for further monitoring and visualization.
 
-Independent of the work describe above, early discussions have already yielded the following requirements on the database infrastructure, with more to come as the work progresses:
+Early discussions have already yielded the following requirements for the database infrastructure:
 
--  Users require a framework/method that manages the point(s) of access, analogous to the EFD Client (`FAFF-REQ-0055`_).
-   Ideally, users will have the impression all queries are going to a single database, despite what is actually happening on the back-end(s).
-- The database must be available and rapidly synced to at all major data facilities (`FAFF-REQ-0055`_), analogous to what is done for the summit EFD.
+- Users require a framework/method that manages the point(s) of access, analogous to the EFD Client (`FAFF-REQ-0055`_).
+  Ideally, users will have the impression all queries are going to a single database, despite what is actually happening on the back-end(s).
+- The database must be available and rapidly synced to all major data facilities (`FAFF-REQ-0055`_), analogous to what is done for the summit EFD.
```

Comment:
As we discussed on zoom, we only need the contents of the DB to be available.  There's no need to sync the data; it's different from EFD as it's derived quantities.

Reply:
Agreed, and I don't think it is necessary to repeat the requirements here, so I propose writing the following for now:
"Early discussions have yielded data access requirements relevant to the database infrastructure (see Data Access)."
and once we converge on the database section, we review the generated requirements. How does that sound?